docs/readme.md

```diff
@@ -0,0 +1,61 @@
+
+The project requires that all contributors follow [Mozilla's code of conduct and etiquette guidelines](/CODE_OF_CONDUCT.md).
+
+This project has [a formal governance document](/GOVERNANCE.md), which describes how various types of contributors work within the project and how decisions are made.
+
+The repository is made available under the terms the [Creative Commons CC0 Public Domain Dedication](/LICENSE). Any contributions must be compatible with its terms. If you're not sure about that, [please ask](#getting-help).
+
+## Ways to contribute
+
+There are many ways you can help improve this repository! For example:
+
+* **Add new compat data**: familiarize yourself with the [schema](https://github.com/mdn/browser-compat-data/blob/master/schemas/compat-data.schema.json) and read the [schema docs](https://github.com/mdn/browser-compat-data/blob/master/schemas/compat-data-schema.md) to add new files.
+* **Fix existing compat data**: maybe a browser now supports a certain feature. Yay! If you open a PR to fix a browser's data, it would be most helpful if you include a link to a bug report or similar so that we can double-check the good news.
+* **Fix a bug:** we have a list of [issues](https://github.com/mdn/browser-compat-data/issues),
+or maybe you found your own.
+* **Review a pull request:** there is a list of [PRs](https://github.com/mdn/browser-compat-data/pulls).
+Let us know if these look good to you.
+
+## Updating compatibility tables on MDN
+
+It takes 1-2 weeks for changes in this data to be reflected in MDN's browser compatibility tables. The process is:
+
+1. A pull request is reviewed and merged to master.
+2. A new release of [mdn-browser-compat-data](https://www.npmjs.com/package/mdn-browser-compat-data) is created by MDN staff. This happens every 4-14 days.
+3. A new image of [Kumascript](https://github.com/mdn/kumascript), which includes the BCD release, is built and deployed to production. This happens within a day of the npm package release.
+4. The MDN page using the data is regenerated. For newly converted pages, a staff member switches to the [{{Compat}}](https://github.com/mdn/kumascript/blob/master/macros/Compat.ejs) macro, and re-checks the conversion. For updates to converted pages, a logged-in MDN user [force-refreshes the page](https://en.wikipedia.org/wiki/Wikipedia:Bypass_your_cache#Bypassing_cache) to regenerate it.
+
+## Opening issues and pull requests
+
+Before submitting your pull request, [validate your new data against the schema](testing.md).
+
+Not everything is enforced or validated by the schema. A few things to pay attention to:
+
+* Feature identifiers (the data namespaces, like `css.properties.background`) should make sense and are spelled correctly.
+* Nesting of feature identifiers should make sense.
+* Notes use correct grammar and spelling. They should be complete sentences ending with a period.
+
+### Generating data using the Web API Confluence Dashboard
```

Comment:
In the past, it was unclear that this step is optional. Maybe mark it as such? "Optional: Generating data using..."

Reply:
Good idea. Done.

Reply:
Awesome, thanks for doing that in here real quick. I will note other improvements in the parent issue (it was just too tempting to get into some of this while reading through the material. Sorry for the extra drive-by work.)

I also realized drive-by edits are nasty :)  You'll need to update the TOC link accordingly now as well.

Reply:
Ha, I totally understand the impulse. It took a lot to not rewrite _everything_ as I went, but I thought it might be easier to review section by section.

I fixed the ToC problem :-)